front-end/src/components/Dashboard/index.js

```diff
@@ -43,21 +50,97 @@ class Dashboard extends React.Component {
   }
 
 
+  hasGH = () =>{
+    if (this.state.github_info === null){
+      return false;
+    } else {
+      return true;
+    }
```

Comment:
You could replace this block with `return this.state.github_info !== null`

Reply:
I'd also rename the function to "hasGitHubInfo". Imagining if a new dev takes over that "hasGH" would be somewhat confusing.